lib/third-party-libs.js.flow

```diff
@@ -2,12 +2,11 @@
  * Basic declarations for the npm modules we use.
  */
 
-declare module "resolve" {
-  declare export default {
-    (string, {| basedir: string |}, (err: ?Error, res: string) => void): void;
-    sync: (string, {| basedir: string |}) => string;
-  };
-}
+declare var require: {
```

Comment:
Do we still need this as `require` is Node.js builtin package?

Reply:
Yeah, flow doesn't support it.